src/js/themes/base.d.ts

```diff
@@ -612,6 +614,9 @@ export interface ThemeType {
     body?: {
       extend?: ExtendType;
     };
+    container?: {
```

Comment:
The container should also be able to accept Box props if I understood correctly.

Reply:
Done.